test/support/fake_record.rb

```diff
@@ -88,6 +88,10 @@ def quote(thing, column = nil)
         "'#{thing.to_s.gsub("'", "\\\\'")}'"
       end
     end
+
+    def cast_bound_value(value)
```

Comment:
why using this fake record class rather than any record?

Reply:
I didn't see any record, what do you suggest?  It looks like the tests don't require a database so my guess is the fake record was used to avoid a db being required for tests but I could be missing something.

Reply:
Oh you are right ! And could we use one method deeper in the backtrace rather than `#sanitize_sql` and not need this fake record while still testing properly ?